src/main/java/cceclipseplugin/ui/ProjectsListView.java

```diff
@@ -103,7 +103,8 @@ private void initButtonListeners() {
 
 			@Override
 			public void handleEvent(Event arg0) {
-				AddProjectDialog dialog = new AddProjectDialog(new Shell());
+                Shell shell = Display.getDefault().getActiveShell();
```

Comment:
This indentation also looks weird. It looks like it is indented left of `public void ...`